rose/server/score_test.py

```diff
@@ -87,10 +87,10 @@ class TestPenguin(SinglePlayerTest):
     def test_pickup(self):
         self.player.action = actions.PICKUP
         self.process()
-        # Player move up and get more life
+        # Player move up and get more score
         assert self.player.x == self.x
         assert self.player.y == self.y - 1
-        assert self.player.life == self.life + 1
+        assert self.player.score == self.score + config.score_penguin_catch + config.move_forward
```

Comment:
Looks like it works like that because you changed it in this patch. If you remove everything which is not related to replacing life with score, will this test be the same?